packages/@react-aria/aria-modal-polyfill/test/index.test.tsx

```diff
@@ -10,18 +10,22 @@
  * governing permissions and limitations under the License.
  */
 
-import {act, fireEvent, render, triggerPress, waitFor} from '@react-spectrum/test-utils';
+import {act, pointerMap, render, simulateMobile, waitFor} from '@react-spectrum/test-utils-internal';
 import {ActionButton, Button} from '@react-spectrum/button';
 import {Content} from '@react-spectrum/view';
 import {Dialog, DialogTrigger} from '@react-spectrum/dialog';
 import {Item, Menu, MenuTrigger, Section} from '@react-spectrum/menu';
 import {Provider} from '@react-spectrum/provider';
 import React from 'react';
 import {theme} from '@react-spectrum/theme-default';
+import userEvent from '@testing-library/user-event';
 import {watchModals} from '../';
 
 describe('watchModals', () => {
+  let user;
+
   beforeAll(() => {
+    user = userEvent.setup({delay: null, pointerMap});
```

Comment:
In some of the tests that didn't have `userEvent` added, I opted to put the userEvent.setup inline with each test case as per https://kentcdodds.com/blog/avoid-nesting-when-youre-testing. Not sure which I prefer tbh...